conftest.py

```diff
@@ -0,0 +1,26 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+# This file is the main file used when running tests with pytest directly,
+# in particular if running e.g. ``pytest docs/``.
+
+import os
+import tempfile
+
+try:
+    from pytest_astropy_header.display import PYTEST_HEADER_MODULES
+except ImportError:
+    PYTEST_HEADER_MODULES = {}
```

Comment:
Need this
```suggestion
try:
    from pytest_astropy_header.display import PYTEST_HEADER_MODULES, TESTED_VERSIONS
except ImportError:
    PYTEST_HEADER_MODULES = {}
    TESTED_VERSIONS = {}

try:
    from pyvo import __version__ as version
except ImportError:
    version = 'unknown'

TESTED_VERSIONS['pyvo'] = version
```

and also this in `setup.cfg` under this section (I think your `[tools:pytest]` is a typo):

```
[tool:pytest]
astropy_header = true
```